app/src/main/res/layout/admin_auth_activity.xml

```diff
@@ -2,64 +2,115 @@
+
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="?attr/colorPrimary"
+        android:fontFamily="sans-serif"
+        android:minHeight="?attr/actionBarSize"
+        android:textSize="20sp"
+        app:title="@string/add_profile_title"
+        app:titleTextColor="@color/white" />
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <FrameLayout
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
-      android:layout_marginTop="16dp"
-      android:paddingStart="32dp"
-      android:paddingEnd="32dp"/>
-    <org.oppia.app.profile.ProfileInputView
-      android:id="@+id/input_pin"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:layout_marginTop="28dp"
-      app:layout_constraintTop_toBottomOf="@id/sub_text"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:label="@string/admin_auth_pin"
-      app:isPasswordInput="true"
-      app:inputLength="5"
-      profile:error="@{viewModel.errorMessage}"/>
-    <Button
-      android:id="@+id/submit_button"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:text="@string/state_continue_submit"
-      style="@style/StateButtonActive"
-      app:layout_constraintTop_toBottomOf="@id/input_pin"
-      app:layout_constraintEnd_toEndOf="parent"
-      android:layout_marginTop="20dp"
-      android:layout_marginEnd="32dp"/>
+      app:layout_constraintTop_toBottomOf="@id/admin_auth_app_bar_layout">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+          android:id="@+id/heading_text"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="40dp"
+          android:fontFamily="sans-serif-medium"
+          android:lines="2"
+          android:paddingStart="28dp"
+          android:paddingEnd="28dp"
+          android:text="@string/admin_auth_heading"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="20sp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+          android:id="@+id/sub_text"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="12dp"
+          android:fontFamily="sans-serif"
+          android:paddingStart="28dp"
+          android:paddingEnd="28dp"
+          android:text="@string/admin_auth_sub"
```

Comment:
Content of this text is different from that in the [mock](https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/a841330e-efe5-4bdb-acdd-4d6e52a59571/PC-MP-Admin-Authorization-Empty-/)

Reply:
@jamesxu0 Can you check this. Basically the the description text as well as text above edit text both are different as compared to mocks. Why is this different? @mschanteltc

Reply:
I think deviating from the original description would be fine, but for the edit text, adding "(5 digits)" would give some inconsistency for other times we ask for the Admin's PIN and didn't specify that it's 5 digits. @seanlip what is your take on this?

Reply:
@mschanteltc -- it's your call, but if you want a data point, IMO we probably don't need the "5 digits" text here.

Reply:
@seanlip I agree, it's best that we omit the "(5 digits)" but we can go ahead and keep the description as it is.